packages/interactive_media_ads/lib/src/android/android_ads_rendering_settings.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import '../platform_interface/platform_interface.dart';
+import 'interactive_media_ads.g.dart' as ima;
+import 'interactive_media_ads_proxy.dart';
+
+/// Android implementation of [PlatformAdsRenderingSettingsCreationParams].
+final class AndroidAdsRenderingSettingsCreationParams
+    extends PlatformAdsRenderingSettingsCreationParams {
+  /// Constructs an [AndroidAdsRenderingSettingsCreationParams].
+  const AndroidAdsRenderingSettingsCreationParams({
+    super.bitrate,
+    super.enablePreloading,
+    super.loadVideoTimeout,
+    super.mimeTypes,
+    super.playAdsAfterTime,
+    super.uiElements,
+    this.enableCustomTabs,
+    @visibleForTesting InteractiveMediaAdsProxy? proxy,
+  })  : _proxy = proxy ?? const InteractiveMediaAdsProxy(),
+        super();
+
+  /// Creates a [AndroidAdsRenderingSettingsCreationParams] from an instance of
+  /// [PlatformAdsRenderingSettingsCreationParams].
+  factory AndroidAdsRenderingSettingsCreationParams.fromPlatformAdsRenderingSettingsCreationParams(
+    PlatformAdsRenderingSettingsCreationParams params, {
+    bool? enableCustomTabs,
+  }) {
+    return AndroidAdsRenderingSettingsCreationParams(
+      bitrate: params.bitrate,
+      enablePreloading: params.enablePreloading,
+      loadVideoTimeout: params.loadVideoTimeout,
+      mimeTypes: params.mimeTypes,
+      playAdsAfterTime: params.playAdsAfterTime,
+      uiElements: params.uiElements,
+      enableCustomTabs: enableCustomTabs,
+    );
+  }
+
+  final InteractiveMediaAdsProxy _proxy;
+
+  /// Notifies the SDK whether to launch the click-through URL using Custom Tabs
+  /// feature.
```

Comment:
My thought process was that `null` essentially meant that the native value will not be set with any value. But since the documentation states that `false` is the default, it is probably safe to always set it and make the default on the Dart side `false`. 

I was actually considering what a better design for setting optional native parameters. Is it better to always set a value if the default is stated or provide an option to not set it (e.g. `null`).